src/Makevars

```diff
@@ -2,4 +2,4 @@
 CXX_STD = CXX11
 
 ## We need headers from our package, the directory is not automatically included
-PKG_CXXFLAGS = -I../inst/include
+PKG_CXXFLAGS = -I../inst/include/nanotime
```

Comment:
Alternatively (and say _a la_ Boost etc) we could change all `#include`.  Better?

Reply:
Yes, hesitated myself here and wasn't sure if there was an R-package idiomatic way. Will modify the headers.

Reply:
I wouldn't call it idiomatic but it sort of grew on me that _standard locations_ like `/usr/include` and alike "just work" and if there are subdirectories (for which there are good reasons) then the subdirs are used in the `#include <...>` statement. Always struck me as most elegant to rely on defaults.